go.mod

```diff
@@ -90,4 +90,4 @@ require (
 )
 
 // Uncomment for local development
-// replace github.com/opslevel/opslevel-go/v2023 => ./submodules/opslevel-go/
+replace github.com/opslevel/opslevel-go/v2023 => ./submodules/opslevel-go/
```

Comment:
Is this because we're not doing a release in advance or should this change!

Reply:
its so the tests will pass.  Upon release we undo this and updated opslevel-go to the released version that happened that day